pkg/config/log.go

```diff
@@ -20,8 +20,19 @@ import (
 	"github.com/cihub/seelog"
 )
 
+// LoggerName is a string prepended to all logs to indicate which component is logging
+type LoggerName string
+
 const logFileMaxSize = 10 * 1024 * 1024         // 10MB
 const logDateFormat = "2006-01-02 15:04:05 MST" // see time.Format for format syntax
+// CoreLogger is the name of the core agent logger
+const CoreLogger LoggerName = "CORE"
```

Comment:
Since these are all the same, maybe the can be grouped in the same `const` declaration? Some other thoughts:

* From an external perspective, reading `config.CoreLogger` I feel like it represents a structure that has special behavior, not a simply a string. Maybe they could be called: `config.LoggerNameCore`, `config.LoggerNameCluster` and `config.LoggerNameDogstatsd`.

Reply:
P.S. prefixing constants like that is quite a common pattern in Go, e.g.: https://golang.org/pkg/net/http/#pkg-constants